pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "crdch_example_workflows"
+version = "0.1.0"
+description = "Example workflows and data for the CRDCH model"
+authors = ["Gaurav Vaidya <gaurav@renci.org>"]
+
+[tool.poetry.dependencies]
+python = ">=3.7.1,<4.0"
+linkml = "^1.1.13"
+jsonschema = "^4.2.1"
+jupyter = "^1.0.0"
+pytest = "^6.2.5"
+PyLD = "^2.0.3"
+pip = "^21.3.1"
+crdch-model = "^1.1.5"
+pandas = "^1.3.4"
+
+[tool.poetry.dev-dependencies]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
Feel free to add:

```
readme = README.md

repository = https://github.com/cancerDHC/example-data
```

under the `[tool.poetry]` section.

Reply:
Fixed in 820153b.